content/influxdb/cloud-dedicated/admin/databases/create.md

```diff
@@ -190,3 +200,22 @@ threshold beyond which query performance may be affected
 
 {{% /expand %}}
 {{< /expand-wrapper >}}
+
+### Custom partitioning
+
+{{< product-name >}} lets you define a custom partitioning strategy for each database.
+A _partition_ is a logical grouping of data stored in [Apache Parquet](https://parquet.apache.org/)
+format in the InfluxDB v3 storage engine. By default, data is partitioned by day,
+but, depending on your schema and workload, customizing the partitioning
+strategy can improve query performance.
+
+Use the `--template-tag` and `--template-time` flags define partition template
+parts used to generate partition keys for the database.
+For more information, see [Manage data partitioning](/influxdb/cloud-dedicated/admin/custom-partitions/).
```

Comment:
Are you going to merge the custom partitioning and storage engine PR as well?

Reply:
Yep. This is going into the `custom-partitions` branch. The next `influxctl` release is going out on Monday or Tuesday. Once that ships, this is ALL going out.